### PR TITLE
Followup fix for guitest

### PR DESCRIPTION
**Pull request by mmattel**
Referencing #329 (update testing doc for oCIS)

The referenced PR removed an attribute making the build fail because no checking was made before merging 👎 

This PR fixes the missing attribute but needs checking if the changes are ok in the context.

Backport to 3.0

### CODE REVIEW

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 1]
```diff
@@ -99,7 +100,8 @@ NOTE: If you are testing against the oCIS server, set *OCIS=true* in the *config
 NOTE: If there are problems with starting Squish, please check the log file `~/.squish/squishlibraryx.log.x`. Also make sure that the `~/squish-for-qt-6.x.x/etc/paths.ini` is user-readable.
 
 ===== Extra Steps for oCIS
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/guitest.adoc:102]:
```suggestion
===== Extra Steps for Infinite Scale
```

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 2]
```diff
@@ -99,7 +100,8 @@ NOTE: If you are testing against the oCIS server, set *OCIS=true* in the *config
...
-While running the tests against oCIS, playwright is used to automate the login process in the browser. So we need to do the following extra steps:
+
+While running the tests against oCIS, playwright is used to automate the login process in the browser. Following extra steps need to be taken:
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/guitest.adoc:104]:
```suggestion
While running the tests against Infinite Scale, playwright is used to automate the login process in the browser. The following extra steps need to be taken:
```

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/appendices/guitest.adoc [hunk 3]
```diff
@@ -122,7 +124,7 @@ sudo docker pull owncloudci/squish
 * xref:run-tests-using-squish[Run Tests Using Squish]
 * xref:run-tests-using-docker[Run Tests Using Docker]
 
-NOTE: Before running middleware, install yarn following the instructions from {yarn-install-url}[here] and clone middleware from {owncloud-test-middleware-url}[here].
+NOTE: Before running middleware, install {yarn-install-url}[yarn] respectively {pnpm-install-url}[pnpm] and clone middleware from {owncloud-test-middleware-url}[here].
```
Comment (saw-jan) [inline modules/ROOT/pages/appendices/guitest.adoc:127]:
```suggestion
NOTE: Before running middleware, install {yarn-install-url}[yarn] and clone middleware from {owncloud-test-middleware-url}[here].
```

**Review by saw-jan — COMMENTED** [review]
We only need nodejs and yarn to run the middleware server.

**Pull request merged**